src/get-started/npm-script.md

```diff
@@ -3,6 +3,38 @@ layout: page
 title: Get Started > NPM Script
 ---
 
-# NPM Script
+# NPM
 
-Need to write up.
+If you already have a build process based on npm scripts you can use `workbox-
+cli` to generate a service worker.
+
+1. Install the module with NPM.
+
+    ```
+    npm install --save-dev workbox-cli
+    ```
+
+1. From your project folder, run `workbox-cli` from a command line. A wizard
```

Comment:
I've moved "From your project folder" to the previous step. 

I'm not seeing what swapping the steps would get me.